### DISCUSSION TITLE
Possible hypermetabolism bugs

### DISCUSSION THREAD

**Discussion by Paperbell** [CleverRaven/Cataclysm-DDA#49111] [General]
There are some things about hypermetabolism that I think are a bit weird.

The easiest to agree on is that hypermetabolism using the health regeneration system allows someone to turn it on for a second at a 5th minute to instantly get the full regeneration for negligible kcal loss. Possible solutions include:
- healing can be made more granular while hypermetabolism is active
- hypermetabolism uses a different healing system
- hypermetabolism needs to be active for at least 5 minutes to give healing

I'm not sure about these next issues:
- medical supplies and good health care skill can boost hypermetabolism to insane levels, allowing you to gain over 100 hit points in each limb every 5 minutes (head heals half as fast). Maybe there could be diminishing returns?
- Hypermetabolism also reduces kcal fast enough to increase weariness intensity by 1 level in less than 10 minutes. Injuries are exhausting to heal, but even with the great healing previously mentioned, going from light to moderate or moderate to very can be worse than escaping with 20 torso health, since it takes a while to reduce weariness. Once you escape danger, food is infinite, but time is as well, so you don't need to use hypermetabolism when you can just sleep. Maybe weariness over time can be reduced slightly compared to actual physical activity?

While it feels unnatural, these last 2 cancel each other out so maybe they can be left as is?

**Comment by Fris0uman** [comment]
> 5th minute to instantly get the full regeneration for negligible kcal loss. 

I need to revisit it, maybe bumping passive healing and metabolims was a bad idea and it should instead explicitly convert calories into hit points over time

**Reply by actual-nh** [reply]
> I need to revisit it, maybe bumping passive healing and metabolims was a bad idea and it should instead explicitly convert calories into hit points over time

Sounds reasonable to me. And perhaps have its rate of healing (and thus caloric consumption) slow down the closer to full (total? worst-injured body part?) HP you are?

**Comment by meelock** [comment]
Maybe instead of functioning as a healing mechanic, hypermetabolism could regenerate weariness while active. As we don't currently have any mechanic other than waiting or sleep to do so.

**Reply by Zeropol** [reply]
Hypermetabolism may instead wear out the player even more because... the metabolism works more.
From [this article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2929498/) : 
>[talk about body size/sleep relation, then] Elephants require only 3 hours of sleep while rats and cats can spend up to 18 hours in sleep. It is postulated that this may be due to differences in metabolism. Smaller animals have higher metabolic rate and higher body and brain temperatures compared to larger animals
